src/lib/components/ui/TextField/README.mdx

```diff
@@ -206,11 +206,9 @@ normalize rendering across browsers.
 HTML attribute **does not limit on how many characters** the user can enter. Use
 the `maxlength` attribute to achieve that effect.
 
-👉 For compatibility with the HTML specification, the `inputSize` option (which
+👉 Note that according to the HTML specification, the `inputSize` option (which
 is translated as `size` attribute of the input element) is
 [not available for the `number` input type](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/number#Controlling_input_size).
-However, you can use the `max` attribute to determine the desired width of the
-number input instead.
 
 <Playground>
```

Comment:
Please just remove the last two `TextField` examples here—we don't want to demonstrate something is *not* working (I believe).
<img width="855" alt="obrazek" src="https://user-images.githubusercontent.com/5614085/121206486-cc0d8180-c878-11eb-9afc-ed209d71975a.png">